src/WhatsNewDialog.cpp

```diff
@@ -127,11 +165,8 @@ END_EVENT_TABLE()
 WhatsNewDialog::WhatsNewDialog(wxWindow* parent, wxWindowID id)
    : wxDialogWrapper(parent, id, XO("Welcome to Audacity!"))
 {
-#if defined(__WXOSX__)
-   SetSize(WindowWidth, 400);
-#else
-   SetSize(FromDIP(wxSize(WindowWidth, 430)));
-#endif
+
+   SetSize(FromDIP(wxSize(WindowWidth, WindowHeight)));
```

Comment:
I just noticed that I have perhaps a logic error here, given that I set the pixel size separately above. Please verify the sizing on macoS

Reply:
Yep, as it is right now, the window is too short, and scrollbars are shown:

![image](https://github.com/user-attachments/assets/98c6fd38-7af7-4242-bf7d-749a7650a23e)

using 460px looks good

![image](https://github.com/user-attachments/assets/71aeaaff-33bd-4a80-b3c3-44a65c37c61f)